Material_FAB/index.ets

```diff
@@ -1,3 +1,17 @@
-export { rippleUtils } from './src/main/ets/components/MainPage/ripple'
-export {FAB_regular} from './src/main/ets/components/MainPage/FAB_regular'
-export {FAB_extended} from './src/main/ets/components/MainPage/FAB_extended'
```

Comment:
package.json update the library name as per the format shared

Reply:
these are the previous changes